pepper-apis/config/application.conf.ctmpl

```diff
@@ -84,7 +84,37 @@
         "cancerLoader": "{{$conf.Data.schedules.cancerLoader}}",
         "studyExport": "{{$conf.Data.schedules.studyExport}}",
         "fileUploadCleanup": "{{$conf.Data.schedules.fileUploadCleanup}}",
-        "tempUserCleanup": "{{$conf.Data.schedules.tempUserCleanup}}"
+        "tempUserCleanup": "{{$conf.Data.schedules.tempUserCleanup}}",
+        "rgpExport": "{{$conf.Data.schedules.rgpExport}}"
+    },
+    "rgpExport": {
```

Comment:
I want to make a note: Pepper is meant to be a general _platform_ for different studies. This configuration is the config file for the Pepper platform. If you look through this file, you won't find mentions of _study-specific_ values. There's no mention of A-T, Angio, Osteo, TestBoston, etc. It concerns me that we have study-specific values (in this case "RGP") in what is meant to be a platform config file. What should have happened is that we come up with a more general feature (e.g. "incremental backup to a bucket") and then these become study-level configuration.

Reply:
This was my initial approach but @zyme suggested I implement this as a one-off.

Reply:
How about if we make a separate config file for the RGP export?

Reply:
Good idea on both counts.  I keep forgetting about secret manager